beyond-essence/inventory/toggle-switches-updated.md

```diff
@@ -55,6 +56,8 @@ On the Manage variations modal, the merchant can switch between two different mo
 * If the toggle is _enabled_: The pen icons on the value bubbles change to x icons allowing the merchant to delete values.
 Also, a confirmation message pops up informing the merchant about the consequences of deleting a variation value.
 
+💡 _Note: This use case is currently not implemented in our software._
```

Comment:
I think we can either go this way or hold off publishing in essence for a few more weeks until this is live.

Reply:
It should go live in the next few weeks, so it's up to you!

Reply:
@sarahwr Will remove the sentence and publish as it is, as the feature will be live very soon and this is only visible internally.